ci-studio-common-util/cmd/root.go

```diff
@@ -0,0 +1,20 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:          "ci-studio-common-util",
+	Short:        "Utility operations to manage the installation of ci-studio-common",
+	SilenceUsage: true,
+}
+
+// Execute does something?
```

Comment:
yup, `Execute()` adds all child commands to the root command and sets flags appropriately.
It should be called by `main.main()` and It only needs to happen once.